src/Component.php

```diff
@@ -129,7 +129,7 @@ protected function loadDefinition(string $pathname) {
   /**
    * Returns the relative file path of the Fractal YAML configuration file for a given component name.
    *
-   * While Fractal do not support file-based variant component configuration files, neither we do.
+   * Like Fractal, this implementation does not support separate configuration files per variant.
```

Comment:
Still exceeds 80 chars

Reply:
Is this a deal breaker? The comment 2 lines prior also exceeds the 80 character limit. I can add line breaks if wanted. 😅

Reply:
The first line of a PHPDoc block is called the PHPDoc summary line.

The PHPDoc summary line must not wrap.